### ISSUE TITLE
Can we use java API in the final exam

### ISSUE THREAD

**Issue by Guo-Yujia** [nus-cs2030/2021-s2#349]
# Summary
I don't know if there is somebody else who aksed this question, but just to double check can we use java API during the exam?

**Comment by gableejh** [comment]
yes you can, you may refer to this page > https://www.comp.nus.edu.sg/~cs2030/exam/
![image](https://user-images.githubusercontent.com/77229491/115999906-69da2380-a620-11eb-84a2-dca518116df1.png)

**Comment by karintang** [comment]
Yes, we can use java API during the exam

**Comment by malcolmsng** [comment]
Yes u can, including code crunch and luminus too

**Comment by ryks4lyfe** [comment]
We are not allowed to use vim right?

**Comment by darrylcjy** [comment]
Yes you can use Java API, LumiNUS and CodeCrunch!

**Comment by rocked10** [comment]
> We are not allowed to use vim right?

We're supposed to write our answer in files so I think we can use any editors to do so

**Comment by luthertan98** [comment]
Yes, we can use the Java API, code crunch, luminus and any folder on your computer

**Comment by Hanyu0118** [comment]
Can Can! 👍

**Comment by chxnhxng** [comment]
yess !!

**Comment by MatthiasTanWeiSheng** [comment]
yep can use api

**Comment by dannytayjy** [comment]
Yup can use API!

**Comment by Martini-Chwang** [comment]
yeapp can use api!

**Comment by Cassianky** [comment]
yes. Remember to open multiple windows beforehand too!
